core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -166,7 +187,7 @@ object JaasTestUtils {
           keyTab = keytabLocation.getOrElse(throw new IllegalArgumentException("Keytab location not specified for GSSAPI")).getAbsolutePath,
           principal = KafkaServerPrincipal,
           debug = true,
-          serviceName = Some("kafka")).toJaasModule
+          serviceName = serviceName).toJaasModule
```

Comment:
Is there a reason for this change? Even with non-IBM module, `serviceName` is optional in the JAAS config, and may be specified as a Kafka property instead.